cmake-docker-environment/labs/subdirectory-example/secondDay/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.9.1)
+project(songTest)
+
+add_library(secondDay secondDay.cpp)
```

Comment:
Given this in the "subdir" example I think it would be nice to have the cpp in a src dir

It's odd seeing an include dir but not src

Reply:
Decided against to avoid convoluting the presentation